### PR TITLE
feat(search): tune search results and add arrow key navigation on search

### PR DESCRIPTION
**Pull request by Drahcirius**
Tiny bit of search result ordering adjustments so that results that have a length similar to the input will be ordered higher.

Added arrow key scrolling to search result dropdown.

### CODE REVIEW

**Comment by iShibi** [comment]
The colour for the arrow key selector is same as one of the colours in the dropdown, due to that it disappears for every alternate option. I would suggest adding either an outline as a selector or a third light colour that highlights the currently selected option. Thanks for adding this btw 👏

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -204,7 +204,16 @@
...
+												index === searchScrollPosition
+													? `
+													ring-1
+													ring-gray-200
+													even:bg-discord-blurple-630
+													dark:even:bg-discord-blurple-660
+													bg-discord-blurple-630
+													dark:bg-discord-blurple-660
+													`
+													: ''
```
Comment (iCrawl) [inline src/components/Navbar.vue:216]: Why isn't this just:
```
index === searchScrollPosition ? 'ring-1 ring-gray-200 even:bg-discord-blurple-630 dark:even:bg-discord-blurple-660 bg-discord-blurple-630 dark:bg-discord-blurple-660' : ''
```

and then let prettier do its formatting?

**Review by iCrawl — CHANGES_REQUESTED** [review]

File: src/components/Navbar.vue (hunk 1 above)
Comment (Drahcirius) [reply src/components/Navbar.vue:216]: I tried, but prettier doesn't even try to attempt to break up this string. It's happy with it being 200 chars long

Comment (iCrawl) [reply src/components/Navbar.vue:216]: It's def better than this 🤣

**Pull request merged**